<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="points_5_Left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="points_10_left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>10 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="points_2_left">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="points_5_right">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="points_10_right">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>10 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="points_2_right">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2 Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_timer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_timer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause_timer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left_Team_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Name</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Timer">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LeftTeamNumber">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Set_time">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Timer</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LeftTeamNumber_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 50pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="digitCount">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Left_Team_name_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Left_team">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>82</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Right_Team">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>82</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScoreboard">
    <property name="title">
     <string>Scoreboard</string>
    </property>
   </widget>
   <addaction name="menuScoreboard"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScore_Settings">
   <property name="text">
    <string>Score Settings</string>
   </property>
  </action>
  <action name="actionTeam_Names">
   <property name="text">
    <string>Team Names</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
